### ISSUE TITLE
Add persistence.conf ?

### ISSUE THREAD

**Issue by patriknw** [akka/akka-microservices-seed-scala.g8#9]
The persistence.conf for Cassandra is not included in the seed because development typically start without the db, like in the tutorial where we first have a dummy gRPC service.

However, we could add a `persistence.conf` but not include it from application.conf (maybe a comment about it there).

**Comment by octonato** [comment]
The annoying part here is that we are tailoring the seed towards some choices. 

I'm fine with gRPC because although not ubiquitous, we consider it our recommendation for microservices. 

While the persistence choices is less obvious. We are already pushing towards Cassandra when we add a docker-compose and the dependencies. 

Maybe we could explore g8 scaffolding http://www.foundweekends.org/giter8/scaffolding.html. 
The drawback is that we need to add another plugin. 

Another easy way to achieve the same is to have some additional stuff on a folder, keeping the same structure. For instance:

```
profiles
└── cassandra
    ├── docker-compose.yml
    └── src
        └── main
            └── resources
                └── persistence.conf
```

The user then do `cp -r profiles/cassandra/* .`

and include the `persistence.conf` in `application.conf` (or uncomment it)

**Comment by patriknw** [comment]
It's too unknown currently to decide what templating engine we need. We might end up with that UI based, we don't know. Therefore we shouldn't do anything fancy here right now. Maybe we scrap g8 also and only provide source zip downloads in the first release. Not sure yet.

**Comment by octonato** [comment]
Makes sense. I also think that those overlays must be done in one shot when configuring it. 

Leaving the option to add more later is too tricky. You can add incompatible stuff or break existing ones already in place. 

And if we remove any tooling, no scaffolding plugin? Just the files that makes sense inside a profile folder. Users can look inside and move then to the right place. Not fancy at all.

**Comment by octonato** [comment]
On the other hand, as it is now, this template is here to support the guide, so makes sense if it is opinionated towards the choices we make in the guide. It makes it a less generic seed template, but that might be good for now.

**Comment by patriknw** [comment]
Seems like [we will not use (remove) this seed](https://github.com/akka/akka-platform-guide/issues/305) until we know what templating approach to use. I'm merging this and syncing it with 00 sources over there.

**Comment by patriknw** [comment]
ah, I thought I was commenting on the PR

**Comment by ignasi35** [comment]
fixed in #11
